app/models/interview.rb

```diff
@@ -8,4 +8,28 @@ class Interview < ActiveRecord::Base
   validates :lastname, :firstname, presence: true
   validates :template_id, presence: true
   validates :target_level, presence: true, inclusion: { in: TARGET_LEVELS, message: "%{value} is not a valid level" }
+
+  def not_answered_first_question
+    first_question = interview_questions[0].id
+    first_question = interview_questions.where('mark IS NULL').first.id if interview_questions.where('mark IS NULL').count > 0
```

Comment:
As I see from method name, it is intended to return a question but it returns an id. Is it OK? 
If so rename it to ```not_answered_first_question_id```

Please follow this rule in future.